packages/backend-common/config.d.ts

```diff
@@ -69,6 +69,16 @@ export interface Config {
        * Defaults to true if unspecified.
        */
       ensureExists?: boolean;
+      /**
+       * Whether plugins should use their own schemas instead of databases. If enabled,
+       * each plugin will create a schema in the configured database instance
+       * using the `pluginId` as its schema name.
+       *
+       * NOTE: Currently only supported by the `pg` client.
+       *
+       * @default false
+       */
+      usePluginSchemas?: boolean;
```

Comment:
Wondering if we could find a better name here, maybe even a value that isn't a boolean?

Thinking stuff like `pluginDivisionMode: 'schemas'`, with it defaulting to `'database'` or whatever makes sense for each driver. The implementation could still be the same though, just a bit of future prep/clarity

Reply:
Updated! I went with `schema` and `database` to stay consistent with the singular nouns, however, I can update those values if you have strong opinions on plural variations!

With my limited context/experience in the project, I couldn't come up with a name better than your suggestion. I'm very happy to update this if you or your team has another suggestion that you like better!

Reply:
👍 , let's :shipit: